src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.test.ts

```diff
@@ -1,33 +1,224 @@
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDidObfuscatedRevocation, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": [Error: revocation block not found for an issuer],
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "reason": Object {
+            "code": 1,
+            "codeString": "UNEXPECTED_ERROR",
+            "message": "revocation block not found for an issuer",
+          },
+          "status": "ERROR",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should fail when revocation is not set to NONE (for now)", async () => {
+      whenPublicKeyResolvesSuccessfully();
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDidCustomRevocation, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": Object {
+            "details": Object {
+              "issuance": Array [
+                Object {
+                  "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+                  "issued": true,
+                },
+              ],
+            },
+            "issuedOnAll": true,
+            "revokedOnAny": true,
+          },
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "status": "INVALID",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should fail when proof is missing", async () => {
+      whenPublicKeyResolvesSuccessfully();
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDidMissingProof, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": [Error: Document is not signed. Proofs are missing.],
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "reason": Object {
+            "code": 1,
+            "codeString": "UNEXPECTED_ERROR",
+            "message": "Document is not signed. Proofs are missing.",
+          },
+          "status": "ERROR",
+          "type": "DOCUMENT_STATUS",
+        }
+      `);
+    });
+    it("should fail when did resolver fails for some reasons", async () => {
+      mockGetPublicKey.mockRejectedValue(new Error("Error from DID resolver"));
+      const res = await OpenAttestationDidSignedDocumentStatus.verify(documentDidSigned, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": Object {
+            "details": Object {
+              "issuance": Array [
+                Object {
+                  "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+                  "issued": false,
+                  "reason": "Error from DID resolver",
+                },
+              ],
+            },
+            "issuedOnAll": false,
+            "revokedOnAny": false,
+          },
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "status": "INVALID",
```

Comment:
this pass ? I expected OpenAttestationDidSignedDocumentStatusCode.UNEXPECTED_ERROR

Reply:
fixed